vendor/README.md

```diff
@@ -113,6 +113,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **EQ:** [atom/language-csharp](https://github.com/atom/language-csharp)
 - **Erlang:** [textmate/erlang.tmbundle](https://github.com/textmate/erlang.tmbundle)
 - **F#:** [fsprojects/atom-fsharp](https://github.com/fsprojects/atom-fsharp)
+- **F*:** [FStarLang/atom-fstar](https://github.com/FStarLang/atom-fstar)
```

Comment:
Wrongly formatted. Needs to be `**F\*:**`, missing backslash here

Reply:
I think it should be enough

Reply:
Or you can use `F✪` which is more beautiful but non-ascii :stuck_out_tongue_winking_eye:

Reply:
Yep, the slash works:


| With an escape: | Without one: |
| ---------------- | --------- |
| *F\* lang*      |   *F* lang*